src/pages/EnablePayments/AdditionalDetailsStep.js

```diff
@@ -271,10 +269,12 @@ class AdditionalDetailsStep extends React.Component {
             );
         }
 
+        const errors = lodashGet(this.props, 'walletAdditionalDetails.errors', {});
         const isErrorVisible = _.size(this.getErrors()) > 0
-            || lodashGet(this.props, 'walletAdditionalDetails.additionalErrorMessage', '').length > 0;
-        const shouldAskForFullSSN = this.props.walletAdditionalDetails.shouldAskForFullSSN;
+            || !_.isEmpty(errors);
+        const shouldAskForFullSSN = this.props.walletAdditionalDetails.errorCode === CONST.WALLET.ERROR.SSN;
         const {firstName, lastName} = PersonalDetails.extractFirstAndLastNameFromAvailableDetails(this.props.currentUserPersonalDetails);
+        const errorMessage = _.isEmpty(errors) ? '' : _.values(errors)[0];
```

Comment:
Don't we need to fetch the last error message in the errors object vs the first one?

Reply:
In our case, it matters very little. Since this is pattern C, there's only ever going to be one error 🤷

Reply:
![Screen Shot 2022-08-02 at 5 37 38 PM](https://user-images.githubusercontent.com/12268372/182388418-9dbef5e7-0f2a-48a7-9695-2fccf1c13d0f.png)

I'm not sure that's true, say a user first received the ssn error and then received the kba error. We'll still show the ssn error.

Reply:
Ah, I guess that's true. Updated